lib/tasks/users.rake

```diff
@@ -149,4 +149,25 @@ namespace :users do
 
     UserPermissionMigrator.migrate(source: source_application, target: target_application)
   end
+
+  desc "Sets 2sv on all users by organisation"
+  task :set_2sv_by_org, [:org] => :environment do |_t, args|
+    organisation = Organisation.find_by(name: args.org)
+    raise "Couldn't find organisation: '#{args.org}'" unless organisation
+
+    users_to_update = User.where(organisation_id: organisation.id, require_2sv: false)
+
+    puts "found #{users_to_update.size} users without 2sv in organsation #{args.org} to set require 2sv flag on"
+
+    users_to_update.each { |user| user.update(require_2sv: true) }
+  end
+
+  desc "Sets 2sv on all users by organisation"
```

Comment:
I assume this is meant to say "by email domain" or similar?

Reply:
Good spot 👍